modules/tinymce/src/core/main/ts/newline/NewLineUtils.ts

```diff
@@ -97,6 +97,9 @@ const getEditableRoot = (dom, node) => {
   return parent !== root ? editableRoot : root;
 };
 
+const isBlockQuote = (editor: Editor) =>
```

Comment:
This doesn't look to be used at anymore? Should this still be here?

- [ ] Consider removing this unused code